blockchain/blockchain.go

```diff
@@ -353,9 +359,10 @@ func (bc *blockchain) context(ctx context.Context, tipInfoFlag bool) (context.Co
 		protocol.WithBlockchainCtx(
 			ctx,
 			protocol.BlockchainCtx{
-				Tip:          tip,
-				ChainID:      bc.ChainID(),
-				EvmNetworkID: bc.EvmNetworkID(),
+				Tip:                   tip,
+				ChainID:               bc.ChainID(),
+				EvmNetworkID:          bc.EvmNetworkID(),
+				IsDeployerWhitelisted: bc.config.IsDeployerWhitelisted,
```

Comment:
not sure whether it is appropriate to add it to context in this way. btw, if it is added to context, no need to add `IsDeployerWhitelisted` to `Blockchain`

Reply:
add to context is for workingset to use when process actions, add to `Blockchain` is for coreservice to use, as explained above

or what is your suggestion?

Reply:
quote what you said above:
```
it is in blockchain interface for 2 reasons:

workingset needs it when processing actions
here core service use it for whitelist deployer check
```